tests/bench_sizeof_coretypes/tests/01-run.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+
+def testfunc(child):
+    child.expect_exact('[SUCCESS]')
+
+
+if __name__ == "__main__":
+    sys.path.append(os.path.join(os.environ['RIOTTOOLS'], 'testrunner'))
+    from testrunner import run
+    sys.exit(run(testfunc))
```

Comment:
PEP8 says newline at end of file, do we enforce PEP8 standards?

Reply:
there is a newline at the end of the file, isnt there?

by the way, on a quick scan, I could not find this information in PEP8 anywhere :-)